<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QDialog" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BazzarOne</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/Background/window.jpeg);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_Sign_up">
      <attribute name="icon">
       <iconset>
        <normaloff>Background/signup.png</normaloff>Background/signup.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Sign UP</string>
      </attribute>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>516</y>
         <width>332</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/new/prefix1/Background/warning.jpeg);</string>
       </property>
       <property name="text">
        <string>SIGN UP</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>10</y>
         <width>1041</width>
         <height>491</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="font">
           <font>
            <family>Ebrima</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>CREATE ACCOUNT</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_fullname">
             <property name="text">
              <string>Fullname</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_fullname">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/new/prefix1/Background/white.jpg);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_fullnameerror">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>8</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_username">
             <property name="text">
              <string>Username</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_username">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/new/prefix1/Background/white.jpg);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_usernameerror">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>8</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_password">
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_password">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/new/prefix1/Background/white.jpg);</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_passerror">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>8</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_reenterPass">
             <property name="text">
              <string>Re-enter password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_reenterPass">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>12</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/new/prefix1/Background/white.jpg);</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_reentererror">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>8</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_type">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Type of account</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QRadioButton" name="radioButton_personal">
               <property name="font">
                <font>
                 <family>Ebrima</family>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>PERSONAL</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_business">
               <property name="font">
                <font>
                 <family>Ebrima</family>
                 <pointsize>12</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>BUSINESS</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_Business">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Please choose anyone of the given accounts</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_Paddress1_Bcomapanyname">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>OpenHandCursor</cursorShape>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Paddress1_Bcomapanyname">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/new/prefix1/Background/white.jpg);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_email_3">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>OpenHandCursor</cursorShape>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_Paddress2_Bwarehouseaddress">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>OpenHandCursor</cursorShape>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Paddress2_Bwarehouseaddress">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/new/prefix1/Background/white.jpg);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_email_2">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>OpenHandCursor</cursorShape>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_email">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>OpenHandCursor</cursorShape>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_email">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/new/prefix1/Background/white.jpg);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_emailerror">
             <property name="font">
              <font>
               <family>Ebrima</family>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_phone">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>OpenHandCursor</cursorShape>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_phone">
             <property name="font">
              <font>
               <family>Ebrima</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/new/prefix1/Background/white.jpg);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_phoneerror">
             <property name="font">
              <font>
               <family>Ebrima</family>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
